sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/expressions/CodeGenerationSuite.scala

```diff
@@ -98,20 +98,15 @@ class CodeGenerationSuite extends SparkFunSuite with ExpressionEvalHelper {
   }
 
   test("SPARK-18091: split large if expressions into blocks due to JVM code size limit") {
```

Comment:
This test simulates the scenario where large code for components of If expression causes JVM's method code size limit to be hit. So the point of having this test is if it fails before the fix for SPARK-18091 was committed and it passes afterwards.